test/providers/dynect_test.rb

```diff
@@ -40,6 +40,23 @@ def test_build_aaaa_from_api
     assert_equal 60, record.ttl
   end
 
+  def test_build_alias_from_api
+    record = @dyn.send(:build_from_api, {
+      "zone" => 'dns-test.shopify.io',
+      "ttl" => 60,
+      "fqdn" => "alias.dns-test.shopify.io",
+      "record_type" => "ALIAS",
+      "rdata" => {
+        "cname" => "dns-test.shopify.io."
```

Comment:
@es oh, that's correct. See here: https://github.com/Shopify/record_store/blob/master/lib/record_store/record/alias.rb#L5